sympy/assumptions/handlers/common.py

```diff
@@ -58,6 +58,21 @@ def Number(expr, assumptions):
     def NaN(expr, assumptions):
         return True
 
+    @staticmethod
+    def Map(expr, assumps):
+        # Unlike other handler, which implies that the object is commutative
+        # with respect to multiplication, this handler implies that this map
+        # commutes the arguments.
```

Comment:
What does this mean?

Reply:
With symbol `x` and `y`, `Q.commutative(x)` (or `y`) means that `x*y == y*x`.  
With map `f`, `Q.commutative(f)` means that `f(x, y) == f(y,x)`.

However, I am working to change this because since `MapMul` is introduced, `Q.commutative(f)` must be compatible with `2*f == f*2`. Maybe I'll introduce another handler, `Q.commutative_map`.